### PR TITLE
Hopefully uncontroversial change proposals

### PR DESCRIPTION
**Pull request by AljoschaMeyer**
Reviewing the full specification is going to take a while, please wait for a comment that specifically says that I'm done.

### CODE REVIEW

**Comment by AljoschaMeyer** [comment]
This comment specifically says that I'm done.

**Comment by johanbove** [comment]
Also thank you @AljoschaMeyer  for this review! Awesome work.

File: docs/specs/data-spec-es5.mdx [hunk 1]
```diff
@@ -581,16 +579,16 @@ Note that documents are mutable but their path can never change (or it would be
 
 Documents may have arbitrary binary bytes associated with them, referred to as attachments. If a document is associated with an attachment, it MUST have a file extension.
 
-The file extension is there to help applications know how to interpret attachment data. For example, documents with JPEG image attachments should have paths ending in `.jpg`, and documents with MP3 audio attachments should have paths ending in `.mp3`.
+The file extension is there to help applications know how to interpret attachment data. For example, documents with JPEG image attachments SHOULD have paths ending in `.jpg`, and documents with MP3 audio attachments SHOULD have paths ending in `.mp3`.
 
-The file extension MUST be positioned at the end of the path to indicate a document with an attachment:
+The file extension MUST be positioned at the end of the path to indicate a document with attachments:
```
Comment (sgwilym) [inline docs/specs/data-spec-es5.mdx:584]: A document can only refer to a single attachment.

Comment (sgwilym) [reply docs/specs/data-spec-es5.mdx:562]: (I will just merge this and make that change myself)

**Comment by sgwilym** [comment]
Thank you for these edits @AljoschaMeyer!

**Pull request merged**